napari/_qt/_qapp_model/_menus.py

```diff
@@ -3,7 +3,7 @@
 from app_model.backends.qt import QModelMenu
 
 if TYPE_CHECKING:
-    from qtpy.QtWidgets import QWidget
+    from qtpy.QtWidgets import QWidget  # type: ignore[attr-defined]
```

Comment:
All those exists, but are can't be seen by mypy (yet?)

Reply:
Hmm I tried without and it actually did not give me problems when running:
`mypy  --python-version=3.x .\napari\_qt\_qapp_model\_menus.py` with x being 8, 9 or 10

Reply:
I observe in past that mypy on my machine returns different result that one on github actions, even when running `tox -e mypy` 
When you run mypy from dev env it may use all of your libraries. 

@Carreau it does not exist for mypy as we do not install any Qt backend in mypy env:
https://github.com/napari/napari/blob/main/resources/requirements_mypy.in

Maybe we should add pyqt5 (or 6) to list of requirements

Reply:
Ok, I can get something out of adding pyqt6 as a dependency. Can we postpone that to another PR though ? It triggers a bunch of other type errors that I'd prefer to deal with later.

Reply:
Yes. I think that it will interact with so many modules that it need separate PR.

Reply:
I agree as well.